src/electron-runner.js

```diff
@@ -94,6 +94,28 @@ function createWindow() {
 		}
 	} );
 
+	if ( process.platform === 'darwin' ) {
```

Comment:
We probably want to have the `theme-light` class apply by default so that non-Mac clients have the light theme.